src/pages/home/sidebar/OptionRowTitle/index.native.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/forbid-prop-types */
+import React, {memo} from 'react';
+import {
+    Text,
+} from 'react-native';
+import {propTypes, defaultProps} from './OptionRowTitleProps';
+
+
+const OptionRowTitle = ({
+    style,
+    option,
+    numberOfLines,
+}) => (
+    <Text style={style} numberOfLines={numberOfLines}>
+        {option.text}
+    </Text>
+);
+
+OptionRowTitle.propTypes = propTypes;
+OptionRowTitle.defaultProps = defaultProps;
+OptionRowTitle.displayName = 'OptionRowTitle';
+
+export default memo(OptionRowTitle);
```

Comment:
Why is this using memo?

Reply:
I think this was my suggestion ... since it's just displaying a plain text component with no internal state I think it's safe to use memo in this case to prevent unnecessary re-renders, right?

Reply:
I think we should only use memo when there is a clear case to do so. This component won't be re-rendering unless the props change anyway, so I don't see any value it it at all and it can just be a simple functional component.